quantizers/kmeans.py

```diff
@@ -3,15 +3,78 @@
 from sklearn.cluster import KMeans
 
 
-class KmeansQuantizer(Quantizer):
+class LloydMaxQuantizer(Quantizer):
```

Comment:
Is this the vector quantizer? Please name it accordingly. If it is it should go in its own file "scalar_kmeans" and "vector_kmeans".

Reply:
Right now I have both scalar and vector kmeans in the same file as different classes. This makes sense to me, but I can make them different files if that makes more sense to anyone else.